tests/helixprep.proj

```diff
@@ -0,0 +1,152 @@
+<Project ToolsVersion="12.0" DefaultTargets="ArchiveAll" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <UsingTask TaskName="ZipFileCreateFromDirectory" AssemblyFile="$(ToolsDir)\net45\Microsoft.DotNet.Build.Tasks.dll"/>
+  
+  <Import Project="dir.props" />
+  <Import Project="..\dir.props" />
+
+  <PropertyGroup>
+    <DiscoveryDirectory>$(TestWorkingDir)</DiscoveryDirectory>
+    <CoreRootDir Condition="'$(RuntimeID)' == '' ">$(CORE_ROOT)</CoreRootDir>
+    <CoreRootDir Condition="'$(RuntimeID)' != '' ">$(CORE_OVERLAY)</CoreRootDir>
+    <CoreRootName Condition="'$(RuntimeID)' == '' ">Core_Root_Windows_NT-$(__BuildArch)</CoreRootName>
```

Comment:
Right, that's logical, but what about the property group or proj-file makes this window-specific enough to hard code "Windows_NT"?